maintenance_equipment_sequence/tests/test_maintenance_equipment_sequence.py

```diff
@@ -60,26 +60,109 @@ def test_01_maintenance_equipment_sequence(self):
         equipment_1.write({"serial_no": False})
         self.assertEqual(equipment_1.serial_no, "TTC0002")
 
-    def test_02_maintenance_equipment_sequence(self):
-        """Create an equipment category linked to a default sequence with a prefix,
-        and verify that the category correctly inherits the sequence prefix.
-        """
-        # Create a sequence
-        seq_1 = self.sequence_obj.create(
+    def test_02_compute_seq_number_next(self):
```

Comment:
All the changes done in your migration commits to the tests file were actually done in the migration to 17.0 and you are not respecting that authorship because te commit is not included in this migration. https://github.com/OCA/maintenance/commit/d31c7ddb6774bd7a1e462d7ba24443998f73d73f